Neos.Neos/Classes/Command/WorkspaceCommandController.php

```diff
@@ -339,11 +343,12 @@ public function rebaseOutdatedCommand(string $contentRepositoryIdentifier = 'def
     /**
      * Display a list of existing workspaces
      *
+     * @param string $contentRepository The name of the content repository. (Default: 'default')
      * @throws StopCommandException
      */
-    public function listCommand(string $contentRepositoryIdentifier = 'default'): void
+    public function listCommand(string $contentRepository = 'default'): void
     {
-        $contentRepositoryId = ContentRepositoryId::fromString($contentRepositoryIdentifier);
+        $contentRepositoryId = ContentRepositoryId::fromString($contentRepository);
         $contentRepository = $this->contentRepositoryRegistry->get($contentRepositoryId);
```

Comment:
@mhsdesign 

Maybe we should use another name. Because we define `$contentRepository` as a variable here again for accessing the `contentRepositoryRegistry` (Same behavior also in the `SubprocessProjectionCatchUpCommandController` and at other parts in the `WorkspaceCommandController`). But at the same time, we define `$contentRepository` again for the name of the content repository for the CLI command input. This is a bit confusing to read then.

So i would suggest we leave it as it is and just use `$contentRepositoryIdentifier` or `$contentRepositoryName` but we wouldn't really have any benefits from that.

What do you think?

Reply:
We could also change the name of the `$contentRepository` variable that is used to access the `contentRepositoryRegistry` of course 🤔

Reply:
funnily i just thought about this dilemma regarding having a `$workspace` as parameter but calling the actual workspace `$workspaceInstance`. I think there is no silver bullet and the `Instance` suffix is a bit hungarian but will also remind us that only little code should reside in controllers (not too much please) so its really just a problem for the boundary. Its more important that the cli arguments match the api we want. Flows abstraction is just not build with the idea of this in mind.

In other places we even `unset` the original variable but that goes to far imo, phpstan will make sure that we wont call any methods on the string variable already ^^

Reply:
So.. you think we can use `$contentRepository` even if we have two variables of that?

Reply:
> you think we can use $contentRepository even if we have two variables of that?
@crydotsnake We should never re-instantiate the same variable in one scope. That is really error prone.
But AFAIS you already changed it, so your question is obsolete, right?

Reply:
I totally agree with you @bwaidelich !

> But AFAIS you already changed it, so your question is obsolete, right?

Yes 🙂